Mixpanel/Mixpanel.m

```diff
@@ -1144,6 +1173,25 @@ - (NSString *)currentRadio
     return @"";
 }
 
+- (void)optOutTracking
+{
+    [[NSUserDefaults standardUserDefaults] setBool:YES forKey:kMPOptOutFlag];
```

Comment:
we should keep track of all distinct_ids that locally have been flagged as opt-out/in

Reply:
should the flag be reset on `reset`?

Reply:
negative! :)